### ISSUE TITLE
Plugin ready for Homebridge 2?

### ISSUE THREAD

**Issue by booshtukka** [lukasroegner/homebridge-dyson-pure-cool#348]
Request for developer to mark this plugin as ready for HB2, or to do whatever needs to be done to be ready. :)

**Comment by speedingcheetah** [comment]
no. this plugin just crashes when u try to load it. It does not conform to the new HB 2.0 standard. Seems to be an abandoned plugin.

**Comment by speedingcheetah** [comment]
> Request for developer to mark this plugin as ready for HB2, or to do whatever needs to be done to be ready. :)

 this plugin just crashes when u try to load it in HB 2.0 . It does not conform to the new HB 2.0 standard. Seems to be an abandoned plugin. There already is a bug report for the issue, but it bene ignored, along with many other reports.

**Comment by booshtukka** [comment]
That’s a shame, because it works fine.

**Comment by jabrown93** [comment]
Looks like the `package.json` file just needs to be updated for Homebridge to know it's ready

**Comment by jabrown93** [comment]
@lukasroegner Made a PR to fix this and update vulnerable dependencies, similar to what I did on the Hue Sync Box repo

**Comment by lukasroegner** [comment]
Thank you @jabrown93 for the PR, it is now merged and the new version 2.8.3 is available on NPM.
